### PR TITLE
Shorten line lengths (to ~100 characters max per line).

### LINKED ISSUE #101: Introduce a max line length.
Having really long lines makes it hard to review changes being made to the documentation.

I intend to work on a PR to fix this.

### PR DESCRIPTION
**Pull request by teymour-aldridge**
Fixes #101

### CODE REVIEW

File: src/advanced-topics/optimizations.md [hunk 1]
```diff
@@ -6,9 +6,13 @@ description: Make your app faster.
...
 ## neq\_assign
 
-When a component receives props from its parent component, the `change` method is called. This, in addition to allowing you to update the component's state, also allows you to return a `ShouldRender` boolean value that indicates if the component should re-render itself in response to the prop changes.
+When a component receives props from its parent component, the `change` method is called. This, in addition to allowing 
+you to update the component's state, also allows you to return a `ShouldRender` boolean value that indicates if the 
+component should re-render itself in response to the prop changes.
 
-Re-rendering is expensive, and if you can avoid it, you should. As a general rule, you only want to re-render when the props actually changed. The following block of code represents this rule, returning `true` if the props differed from the previous props:
+Re-rendering is expensive, and if you can avoid it, you should. As a general rule, you only want to re-render when the
+ props actually changed. The following block of code represents this rule, returning `true` if the props differed from
```
Comment (jstarry) [inline src/advanced-topics/optimizations.md]: We shouldn't have leading spaces when wrapping

File: src/advanced-topics/optimizations.md [hunk 2]
```diff
@@ -6,9 +6,13 @@ description: Make your app faster.
 
 ## neq\_assign
 
-When a component receives props from its parent component, the `change` method is called. This, in addition to allowing you to update the component's state, also allows you to return a `ShouldRender` boolean value that indicates if the component should re-render itself in response to the prop changes.
+When a component receives props from its parent component, the `change` method is called. This, in addition to allowing
```
Comment (jstarry) [inline src/advanced-topics/optimizations.md]: This line is 119 chars

**Review by jstarry — COMMENTED** [review]
Thanks for the effort!

**Comment by teymour-aldridge** [comment]
@jstarry I've fixed the issues.
Turns out my editor has different settings for Markdown files than it does for other file types and it has a handy vertical line which normally sits at the 100 character mark, but for Markdown it sits at the 120 character mark.

**Pull request merged**